tests/testthat/helper-vdiffr.R

```diff
@@ -1,33 +1,11 @@
-# default is equal to whether NOT_CRAN is true or not
-enable_vdiffr <- identical(Sys.getenv("NOT_CRAN"), "true")
-
-# disable or enable vdiffr based on the state of USE_VDIFFR, if set
-if (identical(Sys.getenv("USE_VDIFFR"), "true")) {
-  enable_vdiffr <- TRUE
-} else if (identical(Sys.getenv("USE_VDIFFR"), "false")) {
-  enable_vdiffr <- FALSE
-}
+# vdiffr ignores failures when
+#   - VDIFFR_RUN_TESTS is "false" (on Travis CI with older versions and dev version of R)
+#   - CI is not set (on CRAN)
 
 # disable vdiffr if version is old
 if (!requireNamespace("vdiffr", quietly = TRUE) ||
   utils::packageVersion("vdiffr") < "0.2.3.9001") {
```

Comment:
Should this be `< "0.3.0"`?

Reply:
I think this `if` can be removed entirely.